_posts/2023-08-07-hiop.md

```diff
@@ -0,0 +1,6 @@
+---
+title: "Milestone in Power Grid Optimization on World’s First Exascale Supercomputer"
+categories: story
+---
+
+As part of the Exascale Computing Project’s ExaSGD project, a team including LLNL researchers ran [HiOp](https://github.com/LLNL/hiop), an open source optimization solver, on 9,000 nodes of Oak Ridge National Laboratory’s Frontier exascale supercomputer. In the largest simulation of its kind to date, Frontier allowed researchers to determine safe and cost-optimal power grid setpoints over 100,000 possible grid failures (also called contingencies) and weather scenarios in just 20 minutes. “Because the list of potential power grid failures is large, this problem is very computationally demanding,” said computational mathematician and principal investigator for LLNL Cosmin Petra. “The goal of this project was to show that the exascale computers are capable of exhaustively solving this problem in a manner that is consistent with current practices that power grid operators have.” [Read more at LLNL News.](https://www.llnl.gov/news/llnl-team-reaches-milestone-power-grid-optimization-worlds-first-exascale-supercomputer)
```

Comment:
```suggestion
As part of the Exascale Computing Project’s [ExaSGD project](https://www.exascaleproject.org/research-project/exasgd/), a team including LLNL researchers ran [HiOp](https://github.com/LLNL/hiop), an open source optimization solver, on 9,000 nodes of Oak Ridge National Laboratory’s Frontier exascale supercomputer. In the largest simulation of its kind to date, Frontier allowed researchers to determine safe and cost-optimal power grid setpoints over 100,000 possible grid failures (also called contingencies) and weather scenarios in just 20 minutes. “Because the list of potential power grid failures is large, this problem is very computationally demanding,” said computational mathematician and principal investigator for LLNL Cosmin Petra. “The goal of this project was to show that the exascale computers are capable of exhaustively solving this problem in a manner that is consistent with current practices that power grid operators have.” [Read more at LLNL News.](https://www.llnl.gov/news/llnl-team-reaches-milestone-power-grid-optimization-worlds-first-exascale-supercomputer)
```